README.md

```diff
@@ -101,6 +101,14 @@ content_source_key = '' # your content source key
 client.list_all_permissions(content_source_key)
 ```
 
+#### Listing all permissions with paging
+
+```ruby
+content_source_key = '' # your content source key
+
+client.list_all_permissions(content_source_key, :current => 2, :size => 40)
```

Comment:
I couldn't make API show more than 25 results per page, so that might not work.

Reply:
https://github.com/elastic/enterprise-search-ruby/pull/15/commits/6cdc18800f2865951da5436e91ebd669f388d10c